client/recipient_test.go

```diff
@@ -80,6 +80,57 @@ func TestRecipientsEmail(t *testing.T) {
 	})
 }
 
+func TestRecipientsCustomWebhook(t *testing.T) {
+	t.Parallel()
+
+	ctx := context.Background()
+	c := newTestClient(t)
+
+	body := "webhook body"
+
+	testCases := []struct {
```

Comment:
we just have the one case for now but I was going to leave this structure we use lots of other places for ease of case additions as we go unless someone is opposed